packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/VSphereVirtualMachinesList.tsx

```diff
@@ -67,7 +67,7 @@ export const vSphereVmFieldsMetadataFactory: ResourceFieldFactory = (t) => [
   },
   {
     resourceFieldId: 'powerState',
-    jsonPath: (data) => getVmPowerState('vsphere', (data as VmData)?.vm),
+    jsonPath: (data) => getVmPowerState((data as VmData)?.vm),
```

Comment:
jsonPath: (data: VmData) => getVmPowerState(data?.vm)
 
 looks nicer to me, can you try ?